tests/ImageSharp.Tests/Formats/Png/PngDecoderTests.Chunks.cs

```diff
@@ -80,17 +84,54 @@ public void Decode_IncorrectCRCForCriticalChunk_ExceptionIsThrown(uint chunkType
         }
 
         [Fact]
-        public void CalculateCrc_Works()
+        public void CalculateCrc_Works_LongerRun()
         {
-            // arrange
-            var data = new byte[] { 0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07 };
-            var crc = new Crc32();
-
-            // act
-            crc.Update(data);
+            // Longer run, enough to require moving the point in SIMD implementation with
```

Comment:
Are there any further corner cases worth of testing?

Reply:
Yup, The current test isn't great, I'll add better reference tests